Traceback (most recent call last):
  File "/eos/user/z/zhenxuan/hhwwgg/DiPhotonJetsBox_M40_2017/job_6/DiPhotonJetsBox_M40_2017_executable_job6.py", line 4, in <module>
    from higgs_dna.analysis import run_analysis
  File "/afs/cern.ch/user/z/zhenxuan/HiggsDNA/higgs_dna/analysis.py", line 11, in <module>
    import awkward
  File "/eos/user/z/zhenxuan/higgs-dna/lib/python3.10/site-packages/awkward/__init__.py", line 16, in <module>
    import awkward._cpu_kernels
  File "/eos/user/z/zhenxuan/higgs-dna/lib/python3.10/site-packages/awkward/_cpu_kernels.py", line 19, in <module>
    lib = ctypes.cdll.LoadLibrary(libpath)
  File "/eos/user/z/zhenxuan/higgs-dna/lib/python3.10/ctypes/__init__.py", line 452, in LoadLibrary
    return self._dlltype(name)
  File "/eos/user/z/zhenxuan/higgs-dna/lib/python3.10/ctypes/__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /eos/user/z/zhenxuan/higgs-dna/lib/python3.10/site-packages/awkward/libawkward-cpu-kernels.so: cannot read file data: Input/output error
